specification.md

```diff
@@ -556,6 +593,32 @@ Represents the competition data related to time and scheduling.
 }
 ```
 
+### Location
```

Comment:
I definitely have to veto this part of the changeset, as it is largely redundant with the "venues" part.

Reply:
More information/thoughts in my general review comment.